src/components/FieldMappingTab.tsx

```
@@ -0,0 +1,1065 @@
+				documentSubType: doc.documentSubType,
+				vcFieldsRaw: doc.vcFields,
+			}));
+			if (Array.isArray(profileMappingData.data.value) && profileMappingData.data.value.length > 0) {
+				const mapped = profileMappingData.data.value.map((item, idx) => ({
+					id: Date.now() + idx,
+					fieldId: fields.find(f => f.fieldId === item.fieldId)?.fieldId || fields.find(f => f.name === item.fieldName)?.fieldId || '',
+					documentMappings: (item.documentMappings || []).map((doc, j) => {
+						// Find the corresponding document config
+						const docConfig = vcConfigDocs.find(d => d.documentSubType === doc.document);
+						let vcFields = [];
+						if (docConfig && docConfig.vcFieldsRaw) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)